### DISCUSSION TITLE
Afet Yönetim Sistemi -- Başladı

### DISCUSSION THREAD

**Discussion by bgunduz55** [acikkaynak/deprem-yardim-projesi#35] [Ideas]
Günüllü olarak  afet bölgesine giden kişilerin vasıflarının, alet edevatının ve mevcut konumlarının bilgileri gibi bilgilerin bulunduğu ve en uygun görevlere atamalarının yapılacağı bir sistem. Bunu dahil edip afad bünyesinde dijital olarak konum bazlı yönlendirmeleri ve yaptıkları iş esnasında işin durumuyla alakalı bilgilerin de güncellenebileceği bir sistem oluşturabiliriz. Afad veya Ahbap bünyesinde kullanılabilir.

Örneğin; Bir STK, tır ayarladı.
Tır şöförünü dahi telefonundan sisteme dahil edecek.
Ne kadar stok var, hangi malzemeler var, ve şu anda nerede hepsi görülebilir.
Ve tır olay yerine yaklaşınca bir konum güncellemesi atılır, senin malzemelerin buraya boşaltılacak diye.

Ya da bir gönüllü, elinde hilti matkap vs. var ve daha önce arama kurtarma yapmış. Bu bilgiler mevcut. Bunu görerek bu kişi uygun bir rol ile enkaz üzerinde çalışan ekibe yönlendirilebilecek.
Hatta gelen sayısına göre afaddan bir kişi yanlarında bulunmasa bile yetkin kişilerden yeni gruplar oluşturulabilir.

Ve dashboard tarzı ana sayfada mevcut stok durumları oranları ve ihtiyaçlar gibi kısımlar görüntülenir. Detaya göre arz ve talepteki farklar çıkarılıp STK'lardan organize yardım talebine dönüştürülebilir.

**Burak GÜNDÜZ#2907**
### **Rollere dahil olmak için yoruma discordID'niz ile dahil olmak istediğiniz rolü yazabilirsiniz**
Roller:
FE x 2
BE x 2
Mobile x 2
Tester
Analyse
UI/UX

**Comment by kayisyaparemin** [comment]
şiddetle tavsiye ediyorum. Sadece şimdi değil tüm zamanlarda bu platform üzerinden her şehirde acil durumda kim ne yapabilir elimizde bir veri olur.

backend kısmında yardımcı olabilirim.

.net core
sql
nosql

**Reply by sercangoger** [reply]
bu platformu şuan bu şekilde inşaa etmemiz çok ama çk iyi olacak. keşke daha erken yapabiliyor olsaydık keşke.

**Reply by Sfidan50** [reply]
Afete ve derecesine göre tahmini ihtiyaç listesi çıkaran ve gönüllüleri ayrı ya da afad ekipleri ile birlikte karma ekipler oluşturan, yönlendiren ve takibinin yapılan bir sistem olması verimi artıracaktır. Ek olarak da ekipler için gerekli numara - bağlantıları da sağlaması güzel olur.

**Comment by Mervech** [comment]
Deprem bölgelerine belli olmamakla birlikte belki 1 sene boyunca destek olmak gerekecek. Yurt geneli bir seferberlik olması gerekir. 3 gün 5 gün demeden. Süreç uzun. Yardımlar ve gönüllüler sahada uygun dağilamiyor. İlerleyen süreçte gönüllü olarak gidecek vatandaşların kaç gün destek olabilir, elinden neler gelir gibi bir yazilim, app veya site yapilsa. Bölümler çok fazla sahra hastanesi, çadır kentler, yardım dağıtım, psikolojik destek, yemek üretim dağıtım. Saha ve enkaz çalışmaları. Kimin elinden neler geliyorsa, kim kaç gün destek verebilir bu bilgiler bir havuz yapilsa buna göre ihtiyaç olan bölgelere bu insanlar gitse daha düzenli destek olunabilir. Salgın ve çeşitli risklerle karşı karşıyayız bunlarida unutmamak gerekir. Gönüllülerin il seçme hakki olmasindan ziyade nerede destek gerekiyorsa oraya yonlendirme yapılması lazim. Sahadaki arkadaşlarımız yavas yavaş geri dönmeye başladı. Yerine yeni ekiplerin gitmesi gerekiyor. Bu süreç tahminimizden uzun olacak. Saha da destek olmak isteyen ama çekinen insanlar içinde bir şevk olabilir. Hepimizin işi ailesi çocuğu vs birşeyler var ama elimde imkani olanlar yetkinliği cesareti olanlar 3 gun demeden böyle bir ağ da buluşabilirse daha iyi yardim organize edilir doye düşünüyorum. Bu temel sorumluluğumuz ülke geneli düzenli bir şekilde olması gerekir. Şu anda giden insanlarin çoğu organizasyon eksikliğinden dolayi çok sıkıntı yaşanıyor. Fikirlerinizi ekleyin lütfen. Hepimize kolay gelsin.

**Comment by ramazansancar** [comment]
Merhaba @bgunduz55,
Legal ekibi ile birlikte projeyi inceledik. Şartlı olarak proje sürdürülebileceği düşüncesindeyiz. Notu sana iletiyorum.
"AFAD ve ilgili kuruluşlarla temasa geçilip proje onayı alınırsa legal zemin oluşur"

**Reply by bgunduz55** [reply]
@ramazansancar Evet hocam önemli olan nokta da bu zaten. Tüm sistemlerle entegre olarak işleri çok düzenli şekilde yürütmeye müsait bir sistem olur. Afad veya Ahbap ile konuyla alakalı irtibata geçip işi hızlandırabilecek kimse var mıdır?

**Reply by bgunduz55** [reply]
Bu konuda destek olabilecek kimse yok mu? @ramazansancar

**Comment by TolgaTaner** [comment]
teknik olarak native düşünülürse iOS kısmında görev alabilirim. Tolga Taner - iOS#3606

**Reply by bgunduz55** [reply]
Native uygun olur. Süreçle alakalı iletişimde olabiliriz. Açıklamaya da kendi ID'mi ekliyorum. Teşekkürler.

**Reply by TolgaTaner** [reply]
uygundur, gözümden kaçması durumunda, hızlı iletişim için linkedIn üzerinden yazarsanız kısa sürede discorda geçerim. Native iOS için gönüllü gerekiyorsa da elimden geleni yaparım. Ben teşekkür ederim. @bgunduz55

**Comment by DID0101** [comment]
merhabalar bende yardimci olmak istiyorum nisil yardim edbilirim react , next js , angular ve express js gibi frameforkleri biliyorum nisil yardim edebilirim

**Reply by bgunduz55** [reply]
React angular veya vue frontend uygun olur. DiscordID ni yazarsan sürecin devamıyla alakalı iletişimde olalım.

**Reply by DID0101** [reply]
eski bi discord hesabimi simdi acdim bu galiba DidarPurliyew#6355

**Comment by canermastan** [comment]
Merhaba Java Spring Framework bilgim var uygun olduğu durumda backend konusunda yardımcı olabilirim

**Reply by bgunduz55** [reply]
Spring Boot Microservice backend uygun olur. Discord id ni yazarsan. Sürecin devamıyla alakalı iletişimde olalım.

**Reply by canermastan** [reply]
Hiç microservice deneyimim olmadı ancak kolay öğrenebileceğimi düşünüyorum çalışmayı çok isterim Leprosy#6710

**Reply by bgunduz55** [reply]
Ona bakarız olmadı seniorlar microservice yapısını ayarlarken. İçinde çalışacak endpointleri yazmak gibi işler de yapabilirsin duruma göre.

**Comment by hiulusoy** [comment]
Selamlar Full Stack dev. olarak bütün süreçlerde uzun vadeli rol alabilirim. 

Discord: hiu#5454

Frontend - Angular(7+)
Mobil - Flutter
Backend - Spring Boot, NodeJS(ExpressJS)
Database - SQL (MySQL), NoSQL (MongoDB, Firebase) 
Cloud - AWS

**Reply by bgunduz55** [reply]
Techstack'in güzel. Legal süreçle alakalı dönüş aldığımızda iletişimde oluruz. Teşekkürler.

**Comment by Aldarionn** [comment]
React Native React.js işinize yararsa ise Aldarion#7777 discorddayım

**Reply by bgunduz55** [reply]
Front-end için React olabilir. Legal süreçle alakalı dönüş aldığımızda iletişimde oluruz. Teşekkürler.

**Comment by halilturkoglucs** [comment]
Backend Java Spring Boot kisminda yardimci olabilirim isterseniz:
halil#0144

**Comment by aaysenur** [comment]
Backend Java Spring Boot kisminda ben de yardimci olabilirim
aaysnur#2063

**Comment by agitrubard** [comment]
Back-End tarafta projenin ve altyapının oluşturulması, sürdürülmesi ve ek olarak proje yönetimi gibi konularda destek olabilirim!
agitrubard#7048

**Comment by TheFallingPlane** [comment]
Java, Spring Boot BE olarak destek olabilirim.

açık yazılım ağı dc: Refik Urhan#2321
global dc: TheFallingPlane#2321

**Comment by yvzslmbkts** [comment]
Backend Java Spring Boot ya da herhangi bir dil ben de yardımcı olabilirim. Microservice tecrübem var. 
Yavuz Selim Bektas#2095

**Comment by bgunduz55** [comment]
@DID0101 @canermastan @hiusanmaz @aaysenur @TheFallingPlane @yvzslmbkts Tüm arkadaşlar. Saat 22:00 gibi Grup-2 de toplanalım proje süreciyle alakalı.

**Reply by canermastan** [reply]
Afet sebebiyle yetiştirmem gereken projelerde süreç değişiklikleri oldu maalesef katılamayacağım bu projeye umarım bir sonraki projelere katılım sağlayabilirim

**Comment by Ruhats** [comment]
Spring boot backend ve micro service deneyimim var ayrica hep full stack calistigim icin angular veya html css ile front end tarafinda yardimci olabilirim

**Reply by bgunduz55** [reply]
Tamamdır DiscordId ni yazar mısın? Bir de 22:00 gibi Grup-2 de görüşelim topluca.

**Reply by Ruhats** [reply]
RuhatSengul#3849

**Comment by Rapter1990** [comment]
Selamlar, Discord grubundaki bu linki yeni gördüm. Discord'daki adım direct_x_34#1331. Backend tarafında Java, Spring Boot ihtiyaç varsa destek olabilirim.

**Comment by halis36** [comment]
Selamlar, PHP (Laravel-Symfony) ihtiyaç varsa destek olabilirim. Ayrıca Kürtçe dil noktasında da destek olabilirim.

**Comment by mfd2144** [comment]
Mobil için  native düşünüyorsunuz yardım etmek isterim bir arkadaş yazmış ama İOS içini ikinci bir yazılımcı gerekirse bu role talibim. Fatih Doğan#8801

**Reply by bgunduz55** [reply]
https://discord.gg/JSYJ4Q4v  buraya gelebilirsin.

**Comment by akaanuzman** [comment]
Mobil tarafında flutter'da yaklaşık 2 yıl tecrübem bulunmaktadır. Bunun harcinde node.js ve firebase tarafında da deneyimlerim mevcuttur. Profilimden repoları inceleyebilirsiniz.
archeva#6297

**Comment by egekaradibak** [comment]
Selam merhaba, full-stack olarak FE,BE veya testing kisminda yardimci olabilirim.
Discord: egekaradibak#8548

Fe: React, Angular, Flutter, Vue, 
BE: Node, Spring, Java, Express

**Comment by simgeco** [comment]
merhaba, proje/urun yonetimi, veri analizi, test kisimlarinda yardimci olabilirim ihtiyac varsa. "dashboard" diye konu arattigimda karsima bu konu cikti :)

**Comment by BasAhmet** [comment]
Selamlar, Discord da grubu yeni gördüm. Discord'daki adım ahmetbas. Tester rolunde yardımcı olmak isterim.

**Comment by batuhan11xyz** [comment]
Merhaba,
Mekatronik Mühendisiyim, hardware alanında bir gereksinim olursa gömülü sistemler ve sistem mühendisliği alanında, software alanında iste Python, ML, DL, Computer vision ve diğer AI alanlarında katkı sağlayabilirim.  Aithusa#6960 ya da aithusa0 kullanıcı adım

**Reply by agitrubard** [reply]
Selamlar @batuhan11xyz,
İlgin için teşekkürler. Sadece Python ile discord botumuz var bunun dışında bu alanlarla ilgili projelerimiz yok fakat projemizin kapsamında eğer bir fikrin varsa, şu projeyi yapabiliriz diyorsan mutlaka https://github.com/afet-yonetim-sistemi organization kısmını inceleyip sürece dahil ol ve bizimle iletişime geç 🚀 

Geri dönüşünü merakla bekliyoruz 🙌🏼
